modules/ROOT/pages/ip-blocklist.adoc

```diff
@@ -0,0 +1,80 @@
+= IP Blocklist
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: IP, blocklist, validation, policy
+
+[width="100%", cols="5,15"]
+|===
+>s| Policy Name | IP Blocklist
+>s|Summary      | Blocks a single IP address or a range of IP addresses from accessing an API endpoint
+>s|Category | Security
+>s|First Mule version available | v3.8.0
+.1+>.^s| Returned Status Codes
+| 403 - IP is rejected
+|===
+
+The IP Blocklist policy controls access to a configured API endpoint from a single IP address or a range of IP addresses. 
+
+The IP Blocklist policy restricts access to a protected resource when a match is found between a source IP (specified when configuring the policy) and a list of individual IPs or range of IPs.
```

Comment:
```suggestion
The IP Blocklist policy restricts access to a protected resource when a match is found between a source IP (specified when configuring the policy) and either a list of individual IPs or a range of IPs.
```

Reply:
I see that the blocklist text is essentially the same as the allowlist text, so please consider the edits to the allowlist section in light of this section as well; I'll stop here.